metrics/SystemMetrics.md

```diff
@@ -7,27 +7,30 @@ TODO: Add list of supported platforms.
+| system/processes/running                  | 1     | Int64 Gauge                | Total number of running processes.                                 | Hostname                |
+| system/processes/blocked                  | 1     | Int64 Gauge                | Total number of blocked processes.                                 | Hostname                |
+| system/memory/total                       | By    | Int64 Gauge                | Total system memory capacity in bytes.                             | Hostname                |
+| system/memory/free                        | By    | Int64 Gauge                | Total free system memory in bytes.                                 | Hostname                |
+| system/memory/used                        | By    | Int64 Gauge                | Total used system memory in bytes.                                 | Hostname                |
+| system/disk/total                         | By    | Int64 Gauge                | The total available system disk space in bytes.                    | Hostname, Volume        |
+| system/disk/used                          | By    | Int64 Gauge                | The total used system disk space in bytes.                         | Hostname, Volume        |
+| process/memory/rss                        | By    | Int64 Gauge                | Resident memory size in bytes.                                     | Hostname                |
+| process/file_descriptor/max               | 1     | Int64 Gauge                | The maximum limit of file descriptor count.                        | Hostname                |
+| process/file_descriptor/open              | 1     | Int64 Gauge                | The number of open file descriptor count.                          | Hostname                |
+| process/heap/total                        | By    | Int64 Gauge                | The process's total allocated heap size in bytes.                  | Hostname                |
+| process/heap/used                         | By    | Int64 Gauge                | The process's total used heap size in bytes.                       | Hostname                |
+| process/cpu/usage                         | s     | Int64 Cumulative           | The total user and system CPU time spent in seconds.               | Hostname                |
 
 ### Labels
 The label keys associated with the above metrics.
 
 | Label Key          | Description                                                                                              |
 |--------------------|----------------------------------------------------------------------------------------------------------|
-| Mode               | The amount of time spent by the CPU in state (i.e. `user, nice, system, idle, iowait`).                  |
+| Mode               | The amount of time spent by the CPU in state (i.e. `user, nice, system, idle, iowait, stolen`).          |
```

Comment:
Since the label value is related to metric it might be better to put that in the label column rather than here.

Reply:
The Label is a combination of key and description, to avoid unnecessary duplication in metrics table I have added separate table for labels. Let me know if you still want me to change.